0000-template.md

```diff
@@ -1,3 +1,14 @@
+<!---
+Copyright <fill in year> The Rust Project Developers. See the COPYRIGHT
```

Comment:
@est31 I *absolutely* agree that we shouldn't maintain an individual list. That's why I'm suggesting just saying "Copyrights are retained by the individual contributors.".  Legally, copyright notices are *not* required (though they must be preserved if included); copyright occurs whether you have a notice or not. But a notice of the form "Copyright YYYY Some Name" is a legally recognized copyright notice, and it implies that the named entity owns the copyright. I'm suggesting avoiding that form, which has a specific legal meaning.

(There's nothing wrong with phrases like "The Rust Developers"; the problem is using them in a copyright notice.)